tests/sparseml/pytorch/models/classification/test_resnet.py

```diff
@@ -63,24 +63,22 @@
         ("resnet50", False, True, resnet50),
         ("resnet50", True, False, resnet50),
         ("resnet50", "base", False, resnet50),
-        ("resnet50", "pruned-conservative", False, resnet50),
-        ("resnet50", "pruned-moderate", False, resnet50),
         ("resnet50_2xwidth", False, True, resnet50_2xwidth),
-        ("resnet50_2xwidth", True, False, resnet50_2xwidth),
```

Comment:
corr or corr = potentially corrupted file. Might need to re-upload. 
torch.load(model.pth) fails from api, ok from downloading straight from the source

Reply:
models re-uploaded from git server, passes tests